engines/suppliers/app/controllers/suppliers/application_controller.rb

```diff
@@ -1,4 +1,16 @@
 module Suppliers
   class ApplicationController < ActionController::Base
+    include ::Shared::Controller::Authentication
+
+    rescue_from 'ActiveRecord::InvalidForeignKey' do |exception|
```

Comment:
Unused block argument - `exception`. You can omit the argument if you don't care about it.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.